diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/files/IndentationRule.kt

```diff
@@ -424,12 +499,63 @@ class IndentationRule(configRules: List<RulesConfig>) : DiktatRule(
 
     companion object {
         private val log = LoggerFactory.getLogger(IndentationRule::class.java)
-        const val INDENT_SIZE = 4
+
+        /**
+         * The default indent size (space characters), configurable via
+         * `indentationSize`.
+         */
+        const val DEFAULT_INDENT_SIZE = 4
         const val NAME_ID = "zct-indentation"
```

Comment:
can you please remove prefix -- now it's not required (we supported ordering by another way)

Reply:
You mean `'s/zst-//g'`, right?

Reply:
I tried, and this change can't be made at this time.

This **does break** the ordering, resulting in multiple test failures in [`org.cqfn.diktat.ruleset.smoke.DiktatSmokeTest`](https://github.com/saveourtool/diktat/blob/master/diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/smoke/DiktatSmokeTest.kt#L51).

Reply:
oh.. will look at it